.changelog/12186.txt

```diff
@@ -0,0 +1,15 @@
+```release-note:improvement
+api: Add support for filtering, sorting, and pagination to the ACL tokens list endpoint
+```
+
+```release-note:improvement
+api: Add support for filtering, sorting, and pagination to the allocations list endpoint
+```
+
+```release-note:improvement
+api: Add support for filtering and pagination to the jobs list endpoint
+```
+
+```release-note:improvement
+api: Add support for filtering and pagination to the volumes list endpoint
+```
```

Comment:
for brevity I think we can condense these into 2: filtering,sorting,pagination for ACLs + allocations, and filtering,pagination for job + volumes